mis_builder/tests/test_aep.py

```diff
@@ -44,6 +44,10 @@ def setUp(self):
                 "account_type": "income",
             }
         )
+        # Set country to companys
+        self.company.country_id = (
+            self.env["res.country"].search([("code", "=", "US")], limit=1).id
+        )
```

Comment:
Can you set `country_id` in the company creation statement at line 27 above instead?

Reply:
Done